### PR TITLE
added hw 5.3 solution

### PR DESCRIPTION
**Pull request by programmer28**

### CODE REVIEW

File: src/main/java/core/basesyntax/FileWork.java [hunk 1]
```diff
@@ -9,6 +18,34 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        String resultLine = "";
+        try {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]: can we use try with resources?

File: src/main/java/core/basesyntax/FileWork.java [hunk 2]
```diff
@@ -9,6 +18,34 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        String resultLine = "";
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]: try to use StringBuilder

File: src/main/java/core/basesyntax/FileWork.java [hunk 3]
```diff
@@ -9,6 +18,34 @@
...
+        } catch (FileNotFoundException e) {
+            System.out.println(e);
+        } catch (IOException e) {
+            System.out.println(e);
+        }
+        resultLine = resultLine.replaceAll("[^(\\sa-zA-Z)]", "");
+        String[] words = resultLine.split(" ");
+        List<String> list = new ArrayList<>();
+        for (String word : words) {
+            if (word.toLowerCase().startsWith("w")) {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]: make "w" constant of a class

Comment (programmer28) [reply src/main/java/core/basesyntax/FileWork.java]: I`ve changed it

**Pull request closed without merging**